src/main/java/com/hms_networks/americas/sc/extensions/system/time/SCTimeUtils.java

```diff
@@ -27,4 +52,96 @@ public static String getTimeZoneName() throws Exception {
     // Get time zone name from system control block
     return sysControlBlock.getItem(SCB_TIME_ZONE_KEY);
   }
+
+  /**
+   * Returns the simple date format representing ISO 8601 format with the time zone as UTC.
+   *
+   * @return simple date format representing ISO 8601 format with the time zone as UTC
+   */
+  public static SimpleDateFormat getIso8601UtcTimeFormat() {
+    if (iso8601UtcTimeFormat == null) {
+      throw new IllegalStateException("The local time has not yet been injected.");
+    }
+
+    return iso8601UtcTimeFormat;
+  }
+
+  /**
+   * Returns the simple date format representing ISO 8601 format with the time zone offset appended
+   * (or Z if the timezone is GMT/UTC).
+   *
+   * @return simple date format representing ISO 8601 format with the time zone offset appended
+   */
+  public static SimpleDateFormat getIso8601LocalTimeFormat() {
+    if (iso8601LocalTimeFormat == null) {
+      throw new IllegalStateException("The local time has not yet been injected.");
+    }
+
+    return iso8601LocalTimeFormat;
+  }
+
+  /**
+   * Returns the identifier for the local time zone. (i.e. -05:00, +02:00, Z, etc.)
+   *
+   * @return identifier for the local time zone
+   */
+  public static String getLocalTimeZoneIdentifier() {
```

Comment:
Not sure if the function definition needs a thows Exception
```java
public static String getLocalTimeZoneIdentifier() throws IllegalStateException {
```

Reply:
Added

Reply:
Finished - method renamed as well